client/web/src/components/login-toggle.tsx

```diff
@@ -106,10 +106,8 @@ function LoginPopoverContent({
     }
     setIsRunningCheck(true)
     fetch(`http://${node.IPv4}:5252/ok`, { mode: "no-cors" })
-      .then((r) => {
-        if (r.ok) {
-          setCanConnectOverTS(true)
-        }
```

Comment:
@willnorris looks like this add was breaking our connection check - reverted it back; If we receive a response back at all, we can be certain that they are at least connected over Tailscale.